orc8r/cloud/go/services/bootstrapper/servicers/registration/registration.go

```diff
@@ -66,9 +67,10 @@ func (r *RegistrationService) Register(c context.Context, request *protos.Regist
 	return res, nil
 }
 
-func RegisterDevice(deviceInfo protos.GatewayDeviceInfo, hwid *protos.AccessGatewayID, challengeKey *protos.ChallengeKey) error {
+func RegisterDevice(deviceInfo *protos.GatewayDeviceInfo, hwid *protos.AccessGatewayID, challengeKey *protos.ChallengeKey) error {
+	deviceInfoCopy := proto.Clone(deviceInfo).(*protos.GatewayDeviceInfo)
```

Comment:
you seem to clone every proto message everywhere. Why? Anything changed for Go 1.18 in this regard?

Reply:
see comment above